coredns/gateway/controller.go

```diff
@@ -112,7 +112,7 @@ func (c *Controller) Start(client dynamic.Interface) error {
 		return errors.New("failed to wait for informer cache to sync")
 	}
 
-	go c.queue.Run(c.stopCh, c.processNextGateway)
+	go c.queue.Run(c.processNextGateway)
```

Comment:
Would've preferred this to be different commit as it is a bit unintutive why we have chang ein gateway controller for endpointslices. Not an issue, just was confused a bit by this.

Reply:
yeah sorry this was due to the change in the admiral PR to remove the `stopCh` param. This was another API change made by that PR.